simphony_metadata.yml

```diff
@@ -259,15 +259,14 @@ CUDS_KEYS:
     CUBA.SURFACE_TENSION:
       default: 0.07
       scope: CUBA.USER
-      shape: (2)
     CUBA.MATERIAL:
       shape: (2)
 
   DIRICHLET_BOUNDARY_CONDITION:
     definition: Dirichlet boundary condition
     parent: CUBA.CONDITION
     models: [CUBA.CONTINUUM]
-    CUBA.VARIABLES:
+    variables:
```

Comment:
'variables' is the 'physical_variables' in https://publicwiki-01.fraunhofer.de/SimPhoNy-Project/index.php/SimPhoNy_Metadata_Schema#Converting_a_general_metadata_element_to_a_class
'variables' are also defined in other CUDS_KEYS (e.g. KS_DFT, CFD, ...).